pkg/web3/sdk_test.go

```diff
@@ -70,6 +73,18 @@ func TestGetBalance(t *testing.T) {
 	t.Logf("Balance: %d\n", balance)
 }
 
+func TestGetLPBalance(t *testing.T) {
```

Comment:
Maybe let's add another test for the case where there the address doesn't have enough LP

Reply:
resolved on the latest commit.